example/sign/lib/pages/basic_page.dart

```diff
@@ -37,7 +38,7 @@ class _BasicPageState extends State<BasicPage>
     with SingleTickerProviderStateMixin {
   bool _initialized = false;
 
-  bool _testnetOnly = false;
+  // bool _testnetOnly = false;
```

Comment:
Is this comment needed / does it need TODO?

Reply:
I'll remove it!